Description: Preserves the Iterators.emptyIterator() method that was removed from the public API in Guava 20.
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: not-needed
--- a/guava/src/com/google/common/collect/Iterators.java
+++ b/guava/src/com/google/common/collect/Iterators.java
@@ -76,7 +76,7 @@
    *
    * <p>The {@link Iterable} equivalent of this method is {@link ImmutableSet#of()}.
    */
-  static <T extends @Nullable Object> UnmodifiableIterator<T> emptyIterator() {
+  public static <T extends @Nullable Object> UnmodifiableIterator<T> emptyIterator() {
     return emptyListIterator();
   }
 
